.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/qri-io/starlib
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:latest
```

Comment:
Do we have a reason to push this up, I was under the impression that we try to keep this at the lowest compatible version (and being explicit about it)

Reply:
it would be nice to keep lowest-possible version, but at this point I'm having too much trouble staying on top of all our repos. Feel like moving to testing against latest is easier, and maybe we can follow up with a lowest-possible CI file?